react/src/component/PerfTestDashboard.jsx

```diff
@@ -50,15 +50,40 @@ const windowsMemoryOptions = [
     { value: 'peakVirtualMemorySize64', label: 'peakVirtualMemorySize64', color: '#FFCC00' },
 ]
 
+
+const iosEnergyOptions = [
+    { value: 'totalCost', label: 'totalCost', color: '#007FFF' },
+    { value: 'cpuCost', label: 'cpuCost', color: '#FFA500' },
+    { value: 'gpuCost', label: 'gpuCost', color: '#8B8970' },
+    { value: 'networkingCost', label: 'networkingCost', color: '#800000' },
+    { value: 'appStateCost', label: 'appStateCost', color: '#FFCC00' },
+    { value: 'locationCost', label: 'locationCost', color: '#808000' },
+    { value: 'thermalStateCost', label: 'thermalStateCost', color: '#4B0080' },
+    { value: 'totalOverhead', label: 'totalOverhead', color: '#8884d8' },
+    { value: 'cpuOverhead', label: 'cpuOverhead', color: '#8B008B' },
+    { value: 'gpuOverhead', label: 'gpuOverhead', color: '#EE7600' },
+    { value: 'networkingOverhead', label: 'networkingOverhead', color: '#CD5C5C' },
+    { value: 'appStateOverhead', label: 'appStateOverhead', color: '#BC8F8F' },
+    { value: 'locationOverhead', label: 'locationOverhead', color: '#8B8B7A' },
+    { value: 'thermalStateOverhead', label: 'thermalStateOverhead', color: '#006400' }
+]
+const iosMemoryOptions = [
+    { value: 'memoryMB', label: 'memoryMB', color: '#007FFF' }
+]
+
 export default class PerfTestDashboard extends React.Component {
     state = {
         perfTestResult: this.props.perfTestResult,
         androidMemoryInfo: undefined,
         androidBatteryInfo: undefined,
         windowsMemoryInfo: undefined,
+        iosEnergyInfo: undefined,
+        iosMemoryInfo: undefined,
         selectedAndroidBatteryOptions: androidBatteryOptions.slice(0, 4),
         selectedAndroidMemoryOptions: androidMemoryOptions.slice(0, 10),
-        selectedWindowsMemoryOptions: windowsMemoryOptions.slice(0, 7)
+        selectedWindowsMemoryOptions: windowsMemoryOptions.slice(0, 7),
+        selectedIosEnergyOptions: iosEnergyOptions.slice(0, 7),
+        selectedIosMemoryOptions: iosMemoryOptions
```

Comment:
If there is only one option, I think we don't need multi-select for iOS memory

Reply:
removed